monitoring/definitions/shared/standard.go

```diff
@@ -86,7 +86,7 @@ func (standardConstructor) Errors(legend string) observableConstructor {
 			return Observable{
 				Name:        fmt.Sprintf("%s_errors_total", options.MetricNameRoot),
 				Description: fmt.Sprintf("%s%s errors every 5m", options.MetricDescriptionRoot, legend),
-				Query:       fmt.Sprintf(`sum%s(increase(src_%s_errors_total{%s}[5m]))`, by, options.MetricNameRoot, filters),
+				Query:       fmt.Sprintf(`sum%s(increase(src_%s_errors_total{%s}[5m])) or on() vector(0)`, by, options.MetricNameRoot, filters),
 				Panel:       monitoring.Panel().LegendFormat(fmt.Sprintf("%s%s errors", legendPrefix, legend)),
```

Comment:
Might I recommend using our panel options library? 😁 This way, we can still have a way of knowing the difference between 0 and missing, but also have nicer cosmetics on the graphs:

```suggestion
				Query:       fmt.Sprintf(`sum%s(increase(src_%s_errors_total{%s}[5m]))`, by, options.MetricNameRoot, filters),
				Panel:       monitoring.Panel().LegendFormat(fmt.Sprintf("%s%s errors", legendPrefix, legend))..With(monitoring.PanelOptions.ZeroIfNoData()),
```

Reply:
I swear I used this _incorrectly_ in the past and it didn't work